src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,202 @@
+        }
+    }
+
+    @Override
+    public Car get(Long id) {
+        String getQuery = "SELECT c.id AS car_id, model, registration_number, "
+                + "m.id AS manufacturer_id, m.name, m.country "
+                + "FROM cars c JOIN manufacturers m ON c.manufacturer_id = m.id "
+                + "WHERE c.id = ? AND c.is_deleted = FALSE";
+        Car car = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getStatement = connection.prepareStatement(getQuery)) {
+            getStatement.setLong(1, id);
+            ResultSet resultSet = getStatement.executeQuery();
+            if (resultSet.next()) {
+                car = parseCarByManufacturer(resultSet);
+            }
+            return car;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Can't get car by id = " + id, throwable);
+        }
+    }
+
+    @Override
+    public List<Car> getAll() {
+        String getAllQuery = "SELECT c.id AS car_id, model, registration_number, "
+                + "m.id AS manufacturer_id, m.name, m.country "
+                + "FROM cars c JOIN manufacturers m ON c.manufacturer_id = m.id "
+                + "WHERE c.is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getAllStatement = connection.prepareStatement(getAllQuery)) {
+            List<Car> cars = new ArrayList<>();
+            ResultSet resultSet = getAllStatement.executeQuery();
+            while (resultSet.next()) {
+                cars.add(parseCarByManufacturer(resultSet));
+            }
+            for (Car car : cars) {
+                car.setDrivers(getAllDriversByCarId(car.getId()));
+            }
+            return cars;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Can't get list of cars!", throwable);
+        }
+    }
+
+    @Override
+    public Car update(Car car) {
+        String updateQuery = "UPDATE cars SET model = ?, registration_number = ?, "
+                + "manufacturer_id = ? "
+                + "WHERE id = ? AND is_deleted = FALSE";
+        int updateRecordsNumber;
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement updateStatement =
+                        connection.prepareStatement(updateQuery)) {
+            updateStatement.setString(1, car.getModel());
+            updateStatement.setString(2, car.getRegistrationNumber());
+            updateStatement.setLong(3, car.getManufacturer().getId());
+            updateStatement.setLong(4, car.getId());
+            updateRecordsNumber = updateStatement.executeUpdate();
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Can't update car: " + car, throwable);
+        }
+        if (updateRecordsNumber > 0) {
+            return car;
```

Comment:
here you have to modify drivers also. First deleteAllDrivers where car_id = car.getID()
then insert new drivers().

Reply:
added methods